### ISSUE TITLE
build failed

### ISSUE THREAD

**Issue by unhandyandy** [supercollider/sc3-plugins#83]
I'm trying to install this on Arch Linux.  I have SC 3.7.1 installed from git (supercollider-git).  But when I try to install the plugins I get the following error (truncated).

`[ 17%] Building CXX object source/CMakeFiles/OteyPianoUGens_supernova.dir/OteyPianoUGens/OteyPiano.cpp.o
In file included from /tmp/pacaurtmp-dabrowsa/sc3-plugins-git/src/sc3-plugins/source/OteyPianoUGens/dwgs.h:8:0,
                 from /tmp/pacaurtmp-dabrowsa/sc3-plugins-git/src/sc3-plugins/source/OteyPianoUGens/piano.h:8,
                 from /tmp/pacaurtmp-dabrowsa/sc3-plugins-git/src/sc3-plugins/source/OteyPianoUGens/OteyPiano.cpp:7:
/tmp/pacaurtmp-dabrowsa/sc3-plugins-git/src/sc3-plugins/source/OteyPianoUGens/../DWGUGens/dwglib/DWG.hpp: In member function ‘void TUBE::set_area(float, float)’:
/tmp/pacaurtmp-dabrowsa/sc3-plugins-git/src/sc3-plugins/source/OteyPianoUGens/../DWGUGens/dwglib/DWG.hpp:1271:53: error: no matching function for call to ‘min(float, double)’
             loss = 1.0 - std::min(lossF/sqrt(a1),1.0);
`

**Comment by sonoro1234** [comment]
just made PR #84  for that
@danstowell  somebody has to merge before trying to build again

**Comment by sonoro1234** [comment]
In my mingw gcc 4.8.2 if I change to
```
std::min(lossF/sqrt(a1),1.0f);
```
the complain is for comparing double to float!! (instead of float to double as in your case)

**Comment by danstowell** [comment]
I'd prefer not to receive @ mentions for generic requests to merge a fix for some plugin that I have no relationship with. Better would be for @unhandyandy to test that #84 works and then someone can merge it!

**Comment by sonoro1234** [comment]
I understand but @unhandyandy will not be able to test if we dont merge before

**Comment by bagong** [comment]
The easiest way to checkout a PR I found so far is to type (in the case of your PR):

    git fetch origin pull/84/head:test
    git checkout test

Ready to build...

**Comment by sonoro1234** [comment]
@unhandyandy  Could you try what bagong says or perhaphs you prefer to git clone
```
https://github.com/sonoro1234/sc3-plugins.git
```
and test sc_minchange branch?

**Comment by bagong** [comment]
Or, to avoid having several sc3-plugins repo-folders, you could enter your existing local sc3-plugins repo folder, and:

    git remote add sonoro https://github.com/sonoro1234/sc3-plugins.git
    git fetch sonoro
    git checkout -b sc_minchange sonoro/sc_minchange

;) .r.

**Comment by unhandyandy** [comment]
OK, I followed bagong's instructions to fetch and checkout the test, and it now builds.

**Comment by sonoro1234** [comment]
thanks to everybody :-)

**Comment by bagong** [comment]
Thanks too!

**Comment by danstowell** [comment]
Thanks all, inc @unhandyandy for testing :)
